dotnet/samples/KernelSyntaxExamples/Example44_MultiChatCompletion.cs

```diff
@@ -59,10 +59,9 @@ private static async Task RunChatAsync(IChatCompletion chatCompletion)
         };
 
         // First bot assistant message
-        foreach (IChatResult chatCompletionResult in await chatCompletion.GetChatCompletionsAsync(chatHistory, chatRequestSettings))
+        foreach (var chatContent in await chatCompletion.GetChatContentsAsync(chatHistory, chatRequestSettings))
         {
-            ChatMessage chatMessage = await chatCompletionResult.GetChatMessageAsync();
-            chatHistory.Add(chatMessage);
+            chatHistory.AddMessage(chatContent!);
```

Comment:
chatContent can be null?

Reply:
No, if we ever had a Choice cound == 0 we throw.

https://github.com/microsoft/semantic-kernel/blob/e8797dc264b07932e6d7b20f5337e6af5d20118a/dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ClientCore.cs#L250
```
ClientBase Ln: 241
if (responseData.Choices.Count == 0)
{
    throw new KernelException("Chat completions not found");
}
```